lib/features/base/base_controller.dart

```diff
@@ -14,6 +14,7 @@ abstract class BaseController extends GetxController {
   }
 
   void dispatchState(Either<Failure, Success> newState) {
+    viewState.firstRebuild = true;
```

Comment:
what is it?

Reply:
set a flag to indicate this value is heard

Reply:
so with StreamZip, we don't need this change, am I right?

Reply:
yes.